cypress/e2e/2048-expert/end-game.cy.js

```diff
@@ -0,0 +1,37 @@
+describe('test score intially display correctly', () => {
+    beforeEach(() => {
+        cy.visit("http://localhost:1234/2048-expert/index.html");
+    })
+
+    it('test lost', () => {
+        const TOTALCELL = 36
+        for (let i = 0; i < TOTALCELL; i++){
+            if (i % 2) {
+               cy.get('.cell').eq(i).invoke('text',2) 
+            } else cy.get('.cell').eq(i).invoke('text',4) 
+            
+        }
+
+        cy.get('body').trigger('keyup', { keyCode: 37 })
+
+        cy.on('window:alert', (t) => {
+            //assertions
```

Comment:
nit pick: remove `// assertions` as it is redundant / doesn't add much value for the reader